Add Wi-Fi technology for localization (#16)
ICPS-MicroCity/amusement-park-analysis: doc/sections/2-technologies.tex

          time difference of arrival (TDoA), or propagation model generated from RSSI value.
    \item \textbf{Range-free}: one of the most widely used range-free method is fingerprint localization method. This method can be categorized into three types: visual fingerprint-based localization, motion fingerprint-based systems, and
          signal fingerprint-based methods. In our context the last one is the most promising.
\end{itemize}

\subsubsection{Signal fingerprint-based localization}
Signal fingerprint-based localization is widely used in places where a large number of WiFi infrastructures are deployed. This methods commonly consist of offline training phase
and online fingerprint matching phase. The goal of the first phase is to form a fingerprint database which stores the correlation between \textit{Received Signal Strength} (RSS) from
various \textit{Access Points}(APs) and fix locations. The device's location is determined at the matching stage. In this process we use matching algorithm to search the fingerprint

> in the database which has the minimum difference with the device need to be located and the associated label is our estimated location.

Reviewer: Not sure I get what you mean in this passage

> In terms of effect, signal fingerprint-based localization has the ability to get fine-grained results. However, it is impractical to transfer this method directly from the indoor environment to the

Reviewer: ```suggestion
In terms of effect, signal fingerprint-based localization has the ability to get fine-grained results. However, it is impractical to transfer this method directly from the indoor to the outdoor environment
```

> outdoor environment, because there are many problems such as complicated training work and complex outdoor environment.

Reviewer: ```suggestion
outdoor environment, since there are problems such as complicated training work and complex outdoor conditions.
```
